File: data_monitor/src/main/java/docker_monitor/DM_app/process/object/Threshold.java
package docker_monitor.DM_app.process.object;

public enum Threshold {
    AVERAGE("average"),
    MIN("min"),
    MAX("max"),
    MEDIAN("median");

    private final String val;

    Threshold(String val) {
        this.val = val;
    }
    @Override
    public String toString() {
        return val;
    }
}

File: data_viewer/data_viewer/Model/Notification/Threshold.cs
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace data_viewer.Model.Notification
{
    [JsonConverter(typeof(StringEnumConverter))] 
    public enum Threshold
    {
        [Display(Name = "Average")]
        AVERAGE,
        [Display(Name = "Minimum")]
        MIN,
        [Display(Name = "Maximum")]
        MAX,
        [Display(Name = "Median")]
        MEDIAN

    }
}